src/features/content-answers/ContentAnswers.tsx

```diff
@@ -2,6 +2,10 @@ import React, { useState } from 'react';
 
 import ContentAnswersOpenButton from './ContentAnswersOpenButton';
 
+type ExternalProps = {
+    show?: boolean;
+};
```

Comment:
note for others: this is the approach for ContentOpenWith:
https://github.com/box/box-ui-elements/blob/4d71c6279313b24e9529c9f8b769348453a57226/src/elements/content-open-with/ContentOpenWith.js#L47-L49